api/src/main/resources/application.properties

```diff
@@ -28,3 +28,13 @@ spring.jpa.properties.hibernate.dialect=org.pmiops.workbench.cdr.MySQLDialect
 # and https://precisionmedicineinitiative.atlassian.net/browse/RW-1478
 # Disable all spring-boot endpoints unless specifically necessary
 management.endpoints.enabled-by-default=false
+
+# DB Connection set by environment variables
+spring.datasource.driver-class-name=${DB_DRIVER}
+spring.datasource.url=${DB_CONNECTION_STRING}
+spring.datasource.username=${WORKBENCH_DB_USER}
+spring.datasource.password=${WORKBENCH_DB_PASSWORD}
+cdr.datasource.url=${CDR_DB_CONNECTION_STRING}
+cdr.datasource.username=${CDR_DB_USER}
+cdr.datasource.password=${CDR_DB_PASSWORD}
+server.port=8081
```

Comment:
Probably this should be `${PORT:8081}`: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/appengine-java11/springboot-helloworld/src/main/resources/application.properties - this way it just accepts the port from GAE when its running there.